### ISSUE TITLE
In Some Installs Custom Images Won't Save on the Post Editor

### ISSUE THREAD

**Issue by Nicholas-Cardot** [warfare-plugins/social-warfare#718]
First, we need to try to establish what might be causing this. Here are some basic troubleshooting steps to help us determine the cause. Once we have the cause, a solution should be easy to find.

1. Check for any console errors in the browser.
2. Install Query Monitor and see if any PHP errors, notices, or warnings are present.
3. Enable some sort of error logging on the server or in WordPress, check the error logs after attempting to save them. Check for errors.
4. Check the system status page. Specifically look for the PHP version on the server and the memory allotment that the server has.

**Comment by SquanchyMe** [comment]
Logging here some stats from sites experiencig this:
Ticket 307184 ; PHP 7.0.33, WP Memory 40MB, no console errors on edit page

**Comment by Nicholas-Cardot** [comment]
I wonder if it's a memory issue in that case. 40MB is pretty low. They may want to increase that and then try it. It's likely not our plugin that is eating all that memory, but others using it up and then our fields fail to save would make sense. It's easy to Google some quick tuts on how to increase that. Bump it up to 64MB or 96MB or even 128MB or something and see if it fixes it. Let's try this first to eliminate this as a possibility.

**Comment by SquanchyMe** [comment]
Update: Waiting on 3 sites to increase WP Memory Limit to 128MB to see if that makes any difference

**Comment by SquanchyMe** [comment]
Never heard back from the 3 customers I asked to change their WP Memory Limit.. New ticket today (318291), i just asked them to make the change.

**Comment by SquanchyMe** [comment]
Tracking another (317121), waiting to see if they can change the WP Memory Limit

**Comment by SquanchyMe** [comment]
Internal #318291 - Confirms that increasing WP Memory Limit to 512MB FIXES this problem.
Waiting for more confirmations before closing

**Comment by SquanchyMe** [comment]
Seeing a sudden jump in tickets with this issue - first one checked already has a memory limit of 512MB.. 
Gathering more info.

**Comment by Nicholas-Cardot** [comment]
Closing this as it appears to be a memory issue and increasing it fixes it in all tested cases so far.
